frontend/src/routes/Governance/policies/policy-details/PolicyTemplateDetails.tsx

```diff
@@ -256,7 +256,7 @@ export function PolicyTemplateDetails(props: {
             return ''
           }
           if (cluster && kind && apiVersion && name) {
-            const nameArg = name != '*' ? `&name=${name}` : ''
+            const nameArg = name != '-' ? `&name=${name}` : ''
```

Comment:
Should we disable the View YAML link in the case where the name is a hyphen? I don't think the list of all those resources is particularly helpful, it could be confusing.

Reply:
I think it's useful to see all the objects that are "eligible" for matching the ConfigurationPolicy. Especially if the evaluation interval is long, the result might be out of date but you can still see the search results.

I don't have a strong opinion though, so we could remove it.

Reply:
It would be nice, if the list was reasonably filtered, so that it was stuff that is "eligible" for matching the ConfigurationPolicy. But what we found during testing was that there were several cases where the list was too big (ie all namespaces or all deployments in the cluster), and it was more confusing than useful. I think it is better to remove it for now, and see if we can add it back in specific instances where it is useful.

Reply:
@JustinKuli It's a good point. Would you open an issue on it so the idea doesn't get lost in this thread?